### ISSUE TITLE
Flag_KeepAboveIfNotUtilityWindow (Question)

### ISSUE THREAD

**Issue by nandishnan4** [KDAB/KDDockWidgets#185]
Hi Sergio, 

using the below flags, we can have a Floating window which can be minimized and it remains on top of my parent application. 

flags |= KDDockWidgets::Config::Flag_TitleBarHasMinimizeButton;
flags |= KDDockWidgets::Config::Flag_KeepAboveIfNotUtilityWindow;

However the problem is, the window remains on top even if an external application is clicked.
I understand this is because of Qt::WindowStaysOnTopHint property. 

Is it possible for you to provide a flag, which keeps the window (which can be minimized) always on top of parent application, but not on top of external application.  

Kind Regards
Nandish

**Comment by iamsergio** [comment]
Which Qt flag would this be ?

**Comment by nandishnan4** [comment]
I did not find any Qt Flag that could help me realize this functionality directly.  So I wanted to know if there was a KDAB flag that would help me do this.

**Comment by iamsergio** [comment]
I don't think so, KDDW flags are usually just shortcuts to access Qt's flags.
I'll have another look though.
